diagram-viewer/pom.xml

```diff
@@ -157,5 +161,13 @@
             <artifactId>logback-core</artifactId>
             <version>${logback.version}</version>
         </dependency>
+        <!-- __________________________________________________________________________ -->
+        <!-- Reflection                                                                    -->
+        <!-- __________________________________________________________________________ -->
+        <dependency>
+            <groupId>org.reflections</groupId>
+            <artifactId>reflections</artifactId>
+            <version>0.9.12</version>
+        </dependency>
```

Comment:
we cannot add a library which claims not being under active development (see disclaimer https://github.com/ronmamo/reflections)